experiments/calibration/observationseries_era5.jl

```diff
@@ -0,0 +1,191 @@
+using ClimaAnalysis
+using ClimaLand
+using ClimaLand.Artifacts
+using ClimaCore
+using Statistics
+import EnsembleKalmanProcesses as EKP
+using LinearAlgebra
+using Dates
+FT = Float64
+
+group_by_years(years, data) =
+    Dict(year => data[findall(y -> y == year, years)] for year in unique(years))
+
+# Some variables different conventions/units between ERA5 and CliMA. These
+# preprocessing_function transform everything into the CliMA conventions.
+preprocessing_function = Dict("shf" => (-), "lhf" => (-))
+
+# TODO: This won't be needed when ClimaLand shortvar follow CMIP naming convention
+clima_to_era5 = Dict(
+    "lhf" => "mslhf",
+    "shf" => "msshf",
+    "swu" => "msuwswrf",
+    "lwu" => "msuwlwrf",
+)
+
+function year_from_seasonal_outputvar(out::ClimaAnalysis.OutputVar)
+    # We add one Month because we want to map DJF to the year of JF
+    years = year.(Month(1) .+ dates(out))
+    unique!(years)
+    if length(years) == 1
+        return first(years)
+    else
+        error("Year is not well defined")
+    end
+end
+
+# Load ERA5 data
+era5_path = joinpath(
+    ClimaLand.Artifacts.era5_surface_data_1979_2024_path(),
+    "era5_monthly_averages_surface_single_level_197901-202410.nc",
+)
+
+# Get era5 data. We shift everything to be defined at the beginning of the month
+# so that we can more easily ensure consistency with respect to CliMA data.
+vars_temp = Dict(
+    var_name => OutputVar(
+        era5_path,
+        clima_to_era5[var_name],
+        shift_by = Dates.firstdayofmonth,
+    ) for var_name in variable_list
+)
+
+# Get ClimaLand data (for regridding)
+simdir = SimDir(
```

Comment:
This should not be needed

Reply:
we use it for regridding... is there another way?

```julia
simdir = SimDir( "calibration_output_utki_sample/iteration_000/member_001/global_diagnostics/output_active/",
)

out_vars = Dict(var_name => get(simdir, var_name) for var_name in variable_list)

vars = Dict()
for var in keys(vars_temp)
    preprocessing_fun = get(preprocessing_function, var, identity)
    vars[var] = preprocessing_fun(
        ClimaAnalysis.resampled_as(
            vars_temp[var],
            out_vars[string(var)],
            dim_names = ("longitude", "latitude"),
        ),
    )
end
```

Reply:
Yes, the same way we create the training dataset to begin with.